src/EditorFeatures/Test/Diagnostics/DiagnosticDataSerializerTests.cs

```diff
@@ -31,7 +31,9 @@ public class DiagnosticDataSerializerTests : TestBase
         [Fact, Trait(Traits.Feature, Traits.Features.Diagnostics)]
         public async Task SerializationTest_Document()
         {
-            using var workspace = new TestWorkspace(EditorServicesUtil.ExportProvider, workspaceKind: "DiagnosticDataSerializerTest");
+            using var workspace = new TestWorkspace(composition: EditorTestCompositions.EditorFeatures.WithAdditionalParts(
+                typeof(PersistentStorageServiceFactory)));
```

Comment:
⚠️ `PersistentStorageServiceFactory` is part of the standard composition, so this likely isn't doing what was intended

Reply:
➡️ I see now that this is referring to a different part (down below) with the same name as a part in production code.

Reply:
There is another one defined as a nested type in this class. I'll rename it to TestXxx to avoid confusion.